### PR TITLE
[MM-49023] Add backwards compatibility for board insights activeUsers type

### PR DESCRIPTION
**Pull request by plant99**
#### Summary

Changing `TopBoard.activeUsers` type to array broke insights in community, since we're using an older boards version. This PR adds backwards compatibility to insights.
#### Ticket Link

https://mattermost.atlassian.net/browse/MM-49023
#### Release Note

```release-note
NONE
```

#### QA test steps
 - Webapp master branch crashes when you open insight page with `release-7.5` branch in focalboard
 - This webapp branch shouldn't crash when used with either of the following focalboard branches
     - https://github.com/mattermost/focalboard/blob/main/ - `main` branch
     - https://github.com/mattermost/focalboard/commits/release-7.5 - `release-7.5` branch

### CODE REVIEW

File: packages/types/src/insights.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export type TopBoard = {
     icon: string;
     title: string;
     activityCount: number;
-    activeUsers: Array<UserProfile['id']>;
+    activeUsers: Array<UserProfile['id']> | string;
```
Comment (M-ZubairAhmed) [inline packages/types/src/insights.ts:101]: nit : note here also to mention that for backwards compatibility

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by BenCookie95 — APPROVED** [review]

**Comment by amyblais** [comment]
@furqanmlk Please help prioritize this PR review.

**Review by furqanmlk — APPROVED** [review]
### `Focalboard 7.5.4`

image

### `Main`
image

**Pull request merged**